README-ruRU.md

```diff
@@ -2102,7 +2102,7 @@
     include AnotherModule
 
     # вложенные классы
-    CustomErrorKlass = Class.new(StandardError)
+    CustomErrorClass = Class.new(StandardError)
```

Comment:
It is not an error, it is a good practice to avoid language keywords in identifiers, particularly to write Klass instead of Class.